### PR TITLE
Adding children prop to button Button.tsx

### PR DESCRIPTION
**Pull request by alexreardon**

### CODE REVIEW

File: lib/components/src/Button/Button.tsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ export interface ButtonProps {
   small?: boolean;
   outline?: boolean;
   containsIcon?: boolean;
+  children?: React.ReactNode;
```
Comment (ndelangen) [inline lib/components/src/Button/Button.tsx]: I think we shouldn't do this, in fact I recently learned about:

```js
import React, { ComponentProps } from 'react';

interface Props {
  isLink?: boolean;
  primary?: boolean;
  secondary?: boolean;
  tertiary?: boolean;
  gray?: boolean;
  inForm?: boolean;
  disabled?: boolean;
  small?: boolean;
  outline?: boolean;
  containsIcon?: boolean;
}

export type ButtonProps = ComponentProps<typeof ButtonWrapper>;
```

Comment (ndelangen) [reply lib/components/src/Button/Button.tsx]: Want to use that instead of adding children prop 'manually'?

Comment (alexreardon) [reply lib/components/src/Button/Button.tsx]: I think I have a better solution:

Comment (alexreardon) [reply lib/components/src/Button/Button.tsx]: Actually, looking at the source I think you might be right

Comment (alexreardon) [reply lib/components/src/Button/Button.tsx]: Right now the Button does not know about children, onClick and other standard button props

Comment (alexreardon) [reply lib/components/src/Button/Button.tsx]: On my local I fixed it by adding:
```diff
export declare const Button: React.ForwardRefExoticComponent<ButtonProps & React.RefAttributes<any> 
+ & React.HTMLAttributes<HTMLButtonElement>
> & {
    defaultProps: {
        isLink: boolean;
    };
};
```

Comment (ndelangen) [reply lib/components/src/Button/Button.tsx:16]: `React.HTMLAttributes<HTMLButtonElement>` is included in `styled.button`, the best solution is to pass those types up correctly IMHO.

I'll take any improvement PR though 👍

**Review by ndelangen — APPROVED** [review]

**Pull request merged**